tilecloud_chain/tests/tilegeneration/test-fix.yaml

```diff
@@ -66,6 +66,7 @@ defaults:
         wmts_style: default
         mime_type: image/png
         extension: png
+        params: {}
```

Comment:
Do you relay needs that?
Or it is just to test that's working?

Reply:
This is to test the fix.